packages/components/skeleton/README.mdx

```diff
@@ -0,0 +1,3 @@
+# @contentful/f36-skeleton
+
+This package is part of [Forma 36](https://github.com/contentful/forma-36). See the repo for more details.
```

Comment:
I'm wondering what this README file should contain in the future 🤔 Maybe links to README files of all the skeleton components?

Reply:
Added table of contents section here